### PR TITLE
Fixed issue #18422: Remove Google font usage

### PR DESCRIPTION
**Pull request by gabrieljenik**
- Moved Open Sans out from the uploads folder.
- Made some fonts local

### CODE REVIEW

File: application/config/fonts.php [hunk 1]
```diff
@@ -101,6 +101,15 @@
         ),
     ),
 
+    'font-opensans' => array(
```
Comment (Shnoulle) [inline application/config/fonts.php:104]: Why not adding raleway etc … on the same way ?

File: themes/survey/bootswatch/css/variations/cosmos.min.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700");/*!
+@import url("../../../../../assets/fonts/source-sans-pro.css");/*!
```
Comment (Shnoulle) [inline themes/survey/bootswatch/css/variations/cosmos.min.css:1]: Yep … really don't see how to do it differently …
Don't really like it, but no idea on a cleaner way to do …

**Review by Shnoulle — APPROVED** [review]
No big issue in code.
Just some discussion.

**Comment by Shnoulle** [comment]
Still OK for SurveyTheme boostrap variation :)

**Comment by gabrieljenik** [comment]
Add web safe fonts a s fallback on the email sent when "Reset pwd with mass action"

**Pull request merged**